operator/src/test/java/oracle/kubernetes/operator/helpers/AdminPodHelperTest.java

```diff
@@ -41,7 +41,8 @@
 
 @SuppressWarnings("SameParameterValue")
 public class AdminPodHelperTest extends PodHelperTestBase {
-  private static final String INTERNAL_OPERATOR_CERT_FILE_PARAM = "internalOperatorCert";
+  private static final String INTERNAL_OPERATOR_CERT_FILE_PARAM =
```

Comment:
Do we really want another copy of this pathname? (i.e. RestConfigImpl.java has it too) - should we have a shared constant instead?

Reply:
I agree. Any opinions on an existing interface to put the shared constants or create a new interface?

Reply:
I've refactored out the path names and the reading of the certificate data to TuningParameters to remove duplicate code.  Also easiest to add it to TuningParameters as convenience methods as well as easier for unit test changes.  Please take a look and comment.